### PR TITLE
TD Balance Changes 06302016

### PR DESCRIPTION
**Pull request by AoAGeneral**
These are the following changes for TD:

APC AA Gun projectile speed increase from 1c6 to 2c0.

Flame Tank damage vs light increase from 67 to 100.

Flame Tank explosion damage increase from 100 to 115.

Flame Tank damage spread increase from 341 to 400.

Repair Pad HP increase from 600 to 700.

Artillery Turn Speed increased from 2 to 4.

MCV HP increase from 950 to 1200.

### CODE REVIEW

File: mods/cnc/weapons/smallcaliber.yaml [hunk 1]
```diff
@@ -185,7 +185,7 @@ APCGun.AA:
 	Report: gun20.aud
 	ValidTargets: Air
 	Projectile: Bullet
-		Speed: 1c682
+		Speed: 2
```
Comment (GraionDilach) [inline mods/cnc/weapons/smallcaliber.yaml]: What?!

Unless you actually aimed this to be evaded literally by anything moving faster than a snail, this is wrong.

Comment (GraionDilach) [inline mods/cnc/weapons/smallcaliber.yaml]: Going from the changelog, I assume you wanted `2c0` here, because that would be an increase.

**Comment by AoAGeneral** [comment]
Actually its faster. Doesn't c = Cell location? I tested with this Anjew on the 2 and it hit point target more accurately then 1c682

File: mods/cnc/rules/tech.yaml [hunk 2]
```diff
@@ -1,6 +1,23 @@
+^TechBuildingHusk:
+	Inherits: ^CivBuildingHusk
+	Health:
+		HP: 1
+	DamageMultiplier:
+		Modifier: 0
+	Targetable:
+		RequiresForceFire: yes
+	Capturable:
```
Comment (pchote) [inline mods/cnc/rules/tech.yaml]: From a game design perspective, I think this is a bad idea.  Making dead tech structures repairable removes the incentive and requirement to defend them, and removes the strategy of killing enemy tech structures.

File: mods/cnc/rules/tech.yaml [hunk 3]
```diff
@@ -19,7 +38,7 @@ V19.Husk:
 		Footprint: x
 		Dimensions: 1,1
 	Tooltip:
-		Name: Oil Derrick (Destroyed)
+		Name: Oil Derrick (Recapturable)
```
Comment (pchote) [inline mods/cnc/rules/tech.yaml]: In any case, please do not change the tooltips.  They are descriptive, and it *is* a destroyed oil derrick.

**Comment by AoAGeneral** [comment]
While I can agree with the defending and keeping your tech structures alive, It also costs an engineer to capture the tech structures. Oil Derricks do not refund the 500 so you make the strategy of "Do I take it early and risk unable to defend it? Or do I wait to capture it knowing I can defend it?" It still involves that strategy and also enables late game capture and reuse. Which can also mean holding ground in a designated spot is important rather then keeping to regular expansion trees.

**Comment by GraionDilach** [comment]
I'm all up for rebuildable tech structures.

Denying through demolition still works fine, since from what I seen the tech buildings are usually at arbitrary places which are hard to defend, and engineers are quite expensive as well. What I'd suggest though, is to transfer these building husks to be owned by the Creeps player instead of their original owner with the following:
```
SpawnActorOnDeath:
		OwnerType: InternalName
		InternalOwner: Creeps
```
Not doing this will mean that building husks will get destroyed when the player dies.

**Comment by AoAGeneral** [comment]
Tech Building changes reverted due to some disagreements.

**Comment by GraionDilach** [comment]
This needs to be squashed then.

**Comment by AoAGeneral** [comment]
Whats the command for it? I don't see it in the Git Bash help file or on the contribute page @_@

**Comment by GraionDilach** [comment]
https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history

**Comment by GraionDilach** [comment]
I'm no balance expert but it does what it states, so I guess it's okay to be given my :+1:.

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/dd30e3189d8a66bb9beb1fd65530b8aa58799b3a...8b48fb8cb9a1680e61b748fa69b99c69d33e9b13)

**Pull request merged**